src/main/webapp/dashboards/admin.jsp

```diff
@@ -15,144 +15,161 @@
     <title>Admin Page</title>
 </head>
 <body>
-    <h1>SUPER SECRET ADMIN STUFF</h1>
-    <h2>Audit Log</h2>
-    <ol style="list-style-type: none">
-        <c:forEach items="${requestScope.events}" var="event">
-            <li>${event.createdAt} INFO ${event.description}</li>
-        </c:forEach>
-    </ol>
 
-    <h3>User Account List</h3>
-    <ol style="list-style-type: none">
-        <c:forEach items="${requestScope.users}" var="userAccount">
-            <c:if test="${userAccount.active}">
-                <li>${userAccount.firstName} ${userAccount.surname}</li>
+<div class="outer">
+    <div class="header">
+        <div class="innerDashboard">
+            <img src="${pageContext.request.contextPath}/images/temp_logo.png" alt="logo" id="header-logo">
+            <ul>
+                <li><a href="#">Log Out</a></li>
+                <%-- TODO: Make this work --%>
+            </ul>
+        </div>
+    </div>
+
+    <div class="innerDashboard">
+        <h1 class="title">Admin</h1>
```

Comment:
Code comparison appears to have done a poo on this PR. Likely because I was copying code between two IDE's. Sorry!

All the changes I made on this jsp are above this comment.

Reply:
What happened with the comparison here lol, is it netbeans related? I blame all my problems on netbeans

Reply:
It would be nice to try to revert the changes here to make the commit cleaner in any case